src/main/java/Duke.java

```diff
@@ -1,10 +1,53 @@
+import java.util.Scanner;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.IOException;
```

Comment:
maybe import of java.io should be in front of java.nio/util coz of lexicographic order

Reply:
Will correct it, thanks.